composer.json

```diff
@@ -57,7 +57,9 @@
         "symfony/twig-bridge": "^4.3",
         "symfony/twig-bundle": "^4.3",
         "symfony/validator": "^4.3",
-        "twig/twig": "^2.10"
+        "twig/extra-bundle": "^3.0",
```

Comment:
```suggestion
        "symfony/string": "^5.1",
        "twig/extra-bundle": "^3.0",
```

Reply:
I think it is require like in https://github.com/sonata-project/SonataAdminBundle/pull/6132#discussion_r436359951

Reply:
We should try to be consistent with the decision about this dependency. See https://github.com/sonata-project/SonataIntlBundle/pull/302#discussion_r439797111.

Reply:
> I think it is require like in [sonata-project/SonataAdminBundle#6132 (comment)](https://github.com/sonata-project/SonataAdminBundle/pull/6132#discussion_r436359951)

Not in this case, here we just use the first parameter of the filter (`string|truncate(10)`), so any version of `symfony/string` is valid. In SonataAdmin we use the third one (`cut` parameter previously known as `preserve`) that it's only available in `5.1`.